test/celeritas/ext/RootImporter.test.cc

```diff
@@ -371,3 +370,33 @@ TEST_F(RootImporterTest, volumes)
     EXPECT_VEC_EQ(expected_names, names);
     EXPECT_VEC_EQ(expected_solids, solids);
 }
+
+//---------------------------------------------------------------------------//
+TEST_F(RootImporterTest, em_params)
+{
+    const auto em_params = data_.em_params;
```

Comment:
```suggestion
    const auto& em_params = data_.em_params;
```
You accidentally made a copy :)